spec/spec_helper.rb

```diff
@@ -19,9 +18,7 @@
 # in spec/support/ and its subdirectories.
 Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
 
-# Checks for pending migrations before tests are run.
-# If you are not using ActiveRecord, you can remove this line.
-ActiveRecord::Migration.check_pending! if defined?(ActiveRecord::Migration)
+ActiveRecord::Migration.maintain_test_schema!
```

Comment:
It is unrelated.  I meant to comment about it and ask if it makes sense to submit it as a separate pull request.  I am good either way.  I was going to post the link you just did Rails 4.1 and Rspec 3.0 and forward, the change is the default.  I can't think of a reason not to run a pending migration to the test database automatically, but there could be some.